src/controllers/logToDB.ts

```diff
@@ -7,18 +7,38 @@ export class LogController {
     res: Response
     req: Request
     next: NextFunction
+    
 
     constructor(req: Request, res: Response, next: NextFunction) {
         this.res = res
         this.req = req
         this.next = next
     }
 
+    
+
     async logToDB() {
+        let decoded:TokenInterface;
         const now = new Date()
         const token: string =
             this.req.headers.authorization?.split(' ')[1] || ''
-        const decoded = jwt.verify(token, 'SECRET') as TokenInterface //cambiar secret a .env
+           
+
+            try {
+                decoded = jwt.verify(token, 'SECRET') as TokenInterface
+            }
+            catch{
+                decoded = {userRole:"Guest",userId:"NoUserId"}
+            }
+
+            /*
```

Comment:
Let's remove commented code and run prettier.

Reply:
done